user-guide/Troubleshooting/Procedures/Investigating_Web_Issues.md

```diff
@@ -168,6 +169,35 @@ For example:
 > [!NOTE]
 > When `helper:load-alarms` is set to "false", no alarms will be loaded, even when a visual overview needs alarm information to render correctly.
 
+#### Load balancing
+
+From DataMiner 10.5.2/10.6.0 onwards<!--RN 41434-->, you can implement load balancing for mobile visual overviews among DataMiner Agents in a DMS. Prior to this, the DataMiner Agent to which you were connected would handle all requests and updates with regard to mobile visual overviews.
+
+To configure load balancing:
+
+In the *C:\\Skyline DataMiner\\Webpages\\API\\Web.config* file of a particular DataMiner Agent, add the following keys in the `<appSettings>` section:
+
+- `<add key="visualOverviewLoadBalancer" value="true" />`
+
+  Enables or disables load balancing on the DataMiner Agent in question.
+
+  - When this key is set to "true", for the DataMiner Agent in question, all requests and updates with regard to mobile visual overviews will by default be handled in a balanced manner by all the DataMiner Agents in the cluster.
+
+    However, if you also add the `dmasForLoadBalancer` key (see below), these requests and updates will only be handled by the DataMiner Agents specified in that `dmasForLoadBalancer` key.
+
+  - When this key is set to "false", for the DataMiner Agent in question, all requests and updates with regard to mobile visual overviews will be handled by the local SLHelper process.
```

Comment:
set to "false" or if the key is removed, since no load balancing is the default.

Reply:
```suggestion
  - When this key is set to "false" or is removed, for the DataMiner Agent in question, all requests and updates with regard to mobile visual overviews will be handled by the local SLHelper process.
```